lib/restaurant.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'customer'
+
+class Restaurant
+
+  attr_accessor :dishes
+
+  def initialize
+    @dishes = {
+      :Rice => 1.50,
+      :Naan => 2.00,
+      :Vindaloo => 5.00
+    }
+  end
+
+  def place_order(customer)
+    fail 'Nothing ordered yet' if customer.order.empty?
+    customer.current_order
+    require './send_sms.rb'
```

Comment:
What you need here is a wrapper class for the Twilio gem. Think of a wrapper class as like a phone, and the twilio gem as the SIM card.